langchain4j/src/main/java/dev/langchain4j/service/TypeUtils.java

```diff
@@ -0,0 +1,71 @@
+package dev.langchain4j.service;
+
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+
+public class TypeUtils {

```

Comment:
Could you please cover methods in this class with unit tests?

Reply:
Done...